index.html

```diff
@@ -1,14 +1,39 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="pages/index.css">
+    <meta charset="utf-8" >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" >
+    <link rel="stylesheet" href="pages/index.css" >
     <title>Pink App</title>
   </head>
   <body class="page">
     <div class="page__container">
-      <p>The repository is created with the purpose of learning adaptive web design and team collaboration using Git and GitHub.</p>
+      <p>
+        The repository is created with the purpose of learning adaptive web
+        design and team collaboration using Git and GitHub.
+      </p>
+      <section class="research">
```

Comment:
@Woolfs1423 Great job on adding the hidden heading to improve our page's accessibility! 

Since our page is in russian, all text (even hidden ones), alt, and ARIA attribute values should also be in russian. 

Please replace the hidden heading text here with russian text. Thank you!